components/connectivity-adapter/internal/appregistry/service/handler.go

```diff
@@ -122,41 +159,42 @@ type SuccessfulCreateResponse struct {
 }
 
 func (h *Handler) Get(writer http.ResponseWriter, request *http.Request) {
-	defer h.closeBody(request)
-	gqlCli := h.cliProvider.GQLClient(request)
-
-	id := h.getServiceID(request)
-	gqlRequest := h.gqlRequestBuilder.GetApplicationRequest(id)
-
-	var resp gqlGetApplicationResponse
-	err := gqlCli.Run(context.Background(), gqlRequest, &resp)
-	if err != nil {
-		wrappedErr := errors.Wrap(err, "while getting service")
-		h.logger.Error(wrappedErr)
-		reqerror.WriteError(writer, wrappedErr, apperrors.CodeInternal)
-		return
-	}
-
-	if resp.Result == nil {
-		h.writeErrorNotFound(writer, id)
-		return
-	}
-
-	serviceModel, err := h.converter.GraphQLToDetailsModel(*resp.Result)
-	if err != nil {
-		wrappedErr := errors.Wrap(err, "while converting model")
-		h.logger.Error(wrappedErr)
-		reqerror.WriteError(writer, wrappedErr, apperrors.CodeInternal)
-		return
-	}
-
-	err = json.NewEncoder(writer).Encode(&serviceModel)
-	if err != nil {
-		wrappedErr := errors.Wrap(err, "while encoding response")
-		h.logger.Error(wrappedErr)
-		reqerror.WriteError(writer, wrappedErr, apperrors.CodeInternal)
-		return
-	}
+	//defer h.closeBody(request)
```

Comment:
I know committing commented code is bad, but this is a great base for new handler methods. I can remove it from here if you want to.